src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -32,11 +32,11 @@
 """
 
 WELCOME_MESSAGE = r"""
-    /\\
-   /  \\    _____   _ _ __ ___
-  / /\ \\  |_  / | | | \'__/ _ \\
- / ____ \\  / /| |_| | | |  __/
-/_/    \_\\/___|\__,_|_|  \___|
+     /\\
+    /  \\    _____   _ _  ___ _
+   / /\ \\  |_  / | | | \'__/ _\
+  / ____ \\  / /| |_| | | |  __/
+ /_/    \_\\/___|\__,_|_|  \___|
```

Comment:
The cur version doesn't have two \\ for the A :innocent:

Reply:
Fixed!